README.md

```diff
@@ -16,6 +16,37 @@ Currently, it supports several features:
 
 For a full overview of the commands and flags, run `flink-job-deployer help`
 
+## How to run locally
+
+To be able to test the deployer locally, follow these steps:
+
+1. `go build ./cmd/cli` (build the CLI tool)
+2. `docker-compose build deployer` (build deployer docker container)
```

Comment:
I wonder if you need to include this step. `docker-compose run deployer help` will build it anyway, right?